gotocompany/optimus/integration/v1beta1/event.proto

```diff
@@ -1,6 +1,6 @@
 syntax = "proto3";
 
-package gotocompany.optimus.core.v1beta1;
+package gotocompany.optimus.integration.v1beta1;
```

Comment:
Hey @sudo-suhas, there is a change in the package for events. As well as in the enum.
The start date for the job also added here.